integrations/slack/webhooks/webhook.go

```diff
@@ -195,3 +234,46 @@ func (h handler) dispatchAsyncEventsToConnections(ctx context.Context, cids []sd
 		l.Debug("Event dispatched")
 	}
 }
+
+func (h handler) extractIDs(body []byte, wantContentType string, l *zap.Logger) (appID, teamID, enterpriseID string, err error) {
```

Comment:
I didn't test this yet, but does this really work? Named return values are variables in the function - you need to set their values in the function and call `return` instead of calling `return "x", "y", "z")

...And that's why named return values are usually discouraged, although I understand why you wanted to specify them in this case. Maybe instead document the function?